api/v2/endpoints/post-identity-map.md

```diff
@@ -142,3 +190,12 @@ The following table lists the `status` property values and their HTTP status cod
 | `unauthorized` | 401 | The request did not include a bearer token, included an invalid bearer token, or included a bearer token unauthorized to perform the requested operation. |
 
 If the `status` value is other than `success`, the `message` field provides additional information about the issue.
+
+### Identity Map Policy
+
+The identity map policy let the caller decide when to generate a token. It is passed as an **integer ID** in the request body (with key 'policy') If the parameter is omitted, policy with ID = 0 will be applied.
+
+| ID | Description |
+| :--- | :--- |
+| 0 | Always maps a user identity to UID2. |
+| 1 | Do not map opt-out users. |
```

Comment:
| 1 | Users who have opted out are not included in the mapping. |

Reply:
great, changed!